### PR TITLE
Update changelog for 0.36.0 final 

### PR DESCRIPTION
**Pull request by sklam**
- update release date

### CODE REVIEW

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,4 +1,4 @@
-v0.36.0 (February 04, 2021)
+v0.36.0 (March 11, 2021)
 ---------------------------
```
Comment (stuartarchibald) [inline CHANGE_LOG:2]: Need to alter the number of dashes?

Comment (sklam) [reply CHANGE_LOG:2]: doh

**Comment by esc** [comment]
Would suggest to ignore Public CI since it is obviously glitchin'!

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the patch!

**Pull request merged**